ui/src/components/ListDetails/ListDetails.jsx

```diff
@@ -10,7 +10,6 @@ export default function ListDetails(props) {
     if (props.pic === "") {
         let len = props.subCatRecipes.length
         let randomNum = Math.floor(Math.random() * len);
-        console.log(props.subCatRecipes[randomNum].thumbnailUrl)
         props.subCatRecipes[randomNum].thumbnailUrl === undefined ? 
```

Comment:
curious if we could write this instead:
`
const randomRecipe = props.subCatRecipes[randomNum]
profilePic = randomRecipe.thumbnailUrl === undefined ? randomRecipe.thumbail_url : randomRecipe.thumbnailUrl`

Reply:
just changed it!